microprofile/bean-validation/src/main/java/io/helidon/microprofile/bean/validation/package-info.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2022 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * General Bean Validation.
```

Comment:
The javadoc should explain what this is about, including basic usage (e.g. use this module to enable validation and to map validation exceptions to JAX-RS response).

Reply:
Done